src/GenericPlanForm.tsx

```diff
@@ -4,25 +4,32 @@ import CardContent from '@material-ui/core/CardContent';
 import CardActions from '@material-ui/core/CardActions';
 import IconButton from '@material-ui/core/IconButton';
 import Typography from '@material-ui/core/Typography';
-import { IAllowedPlans, IParameter, ISumbitPlanObject } from './queueserver';
+import { IAllowedPlans, IParameter, ISubmitPlanObject } from './queueserver';
```

Comment:
Ha, good catch! It's a well-hidden typo :)

Reply:
:) 
Thanks for the review!